code/game/objects/structures/tables_racks.dm

```diff
@@ -393,6 +399,7 @@
 			add_logs(user, src, "climbed onto")
 			user.Stun(2)
 			return 1
+			tableclimber = null

```

Comment:
What Gunhog is trying to say is that because this call unsetting tableclimber is after the return, it does not actually get unset, this means that the tableclimber var could potentially remain set and at some later point someone can click on this table and trigger the stun for that user